azure/virtual-machine/examples/basic/main.tf

```diff
@@ -57,7 +59,21 @@ module "vm" {
   tags                = local.tags
 
   computer_name = "vm"
-  password      = var.password
+  password      = "ASfdskjh3&^"
```

Comment:
Could we replace this with a random generator for the example?

Reply:
replaced with var.password